src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,91 @@
+import { shallow } from 'enzyme';
+import React, { Fragment } from 'react';
+
+import { Icon } from '../Icon/Icon';
+
+import { Button } from './Button';
+
+describe('<Button />', () => {
+	it('Should be able to render', () => {
```

Comment:
slechts 1 tests wordt getoond in het storyboard maar jij hebt er hier meerdere:
![image](https://user-images.githubusercontent.com/1710840/58541056-eed11e00-81fa-11e9-87d9-4f92a41d4543.png)

Reply:
Fixed!